.travis.yml

```diff
@@ -23,23 +23,17 @@ jobs:
   include:
     - stage: test
       name: "Unit Tests"
-      script: go test -tags tests ./... -v -cover
-      services:
-        - postgresql
-        - mysql
-      env:
-        - POSTGRES_PASSWORD=""
-        - MYSQL_PASSWORD=""
-        - MYSQL_HOST="127.0.0.1"
+      script:
+      - docker-compose up -d
+      - docker-compose ps
+      - go test -tags tests ./... -v -cover
     - stage: test
       name: "Go Lint"
       script:
         - go get -u golang.org/x/lint/golint
-        - find . -type f -name "*\.go" | grep -v "zz_generated" | while read line; do
+        - find . -type f -name "*\.go" | grep -v "zz_generated" | grep -v "proxy/proxysql.go" | while read line; do
```

Comment:
Um @hyunysmile I think we should perhaps correct the linting rather than just exclude the offending file. We can do this in a later cleanup PR.

Reply:
the reason is that `proxysql` is the name of a tool. not something I invented. so I would prefer to keep as it is. And there is no way to exclude specific file or line with golint

Reply:
If I understand the error message is that `proxy.ProxySQL` stutters. Basically you have the word proxy prepended twice. this type could eventually be simplified to something as simple as `proxy.SQL` which wouldn't stutter at all.